app/views/govuk_publishing_components/components/_government_navigation.html.erb

```diff
@@ -40,8 +40,8 @@
       </a>
     </li>
     <li>
-      <a class="<%= 'active' if active == 'announcements' %>" href="/government/announcements">
-        <%= t("govuk_component.government_navigation.announcements", default: "Announcements") %>
+      <a class="<%= 'active' if active == 'announcements' %>" href="/news-and-communications">
+        <%= t("govuk_component.government_navigation.news_and_communications", default: "News and communications") %>
```

Comment:
Where do these translations come from? I thought it was necessary to update `/config/locales/en.yml` with the text, but it doesn't seem to have the translation this replaces in it either.

Reply:
I don't know where they come from. The default seems to do the trick, and I don't have the welsh translation of this text (yet!! https://trello.com/c/ibFA5dnk/360) so didn't investigate further.

Reply:
Ah, of course, the default. Yeah, that's probably fine then. I've been putting translations into the `en.yml` file on the basis of habit, even though I've never had another language to add. I genuinely don't know what's best in this situation... ?

Reply:
I think we should add text into the language files, even if we only have the english. That way we have a central record of all required translations, so if we have the services of a translator in the future they don't need to hunt through the whole codebase trying to find all the text that needs translating, it's in one place.